rra2json.py

```diff
@@ -187,6 +187,55 @@ def autoassign_rras(config):
     except IndexError:
         debug("No unassigned RRAs")
 
+def autoassign_vas(config):
+    """This will search through unassigned RRA bugs and assign them automatically"""
+    bcfg = config['bugzilla']
+
+    # If no API key has been specified, just skip this
+    if len(bcfg['api_key']) == 0:
+        return
+
+    b = bugzilla.Bugzilla(url=bcfg['url'], api_key=bcfg['api_key'])
+
+    try:
+        with open(bcfg['cache'], 'rb') as f:
+            assign_list = pickle.load(f)
+    except FileNotFoundError:
+        debug("no current autoassign list found, using configured defaults")
+        assign_list = list(bcfg['autoassign'])
+
+    # Do we have any VA in the queue?
+    terms = [{'product': bcfg['product']}, {'component': bcfg['component']},
```

Comment:
I wonder if we it's just easier here to hardcode the components here or have a short-name match in the config, rather than pre-assume it's only RRAs.

Reply:
I suspect name-spacing the config as well would also do the trick, but want feedback if that would be a problem.

Reply:
if we do things "right" we should probably decouple this from rra2json (as well the code rra autoassign functionality), and run it either as a plugin or additional program
if we do things "works for now", just namespace in the configuration